components/report-form.tsx

```diff
@@ -9,22 +12,53 @@ import {
   SelectValue,
 } from "./ui/select";
 import { DateTimePicker } from "./date-time-picker";
-import { FormDescription } from "./ui/form";
 import { Textarea } from "./ui/textarea";
 
 export default function ReportForm() {
+  const [issue, setIssue] = useState("");
+  const [datetime, setDatetime] = useState<Date | null>(null);
+  const [detail, setDetail] = useState("");
+  const router = useRouter();
+
+  const venueId = "920521f5-37a2-46ac-9e03-304763998903";
```

Comment:
get a ID from the DB.

await prisma.venue.findUnique({
     where: {
    // venue.name == "Erich Zeigner Tischtennisplatte" }
})

Reply:
seed script has to be initiated again for changes to apply

venues have a fixed id now to make it work on any machine

react-hook-form implemented

date-time-picker in one window now

date-time-picker automatically sets present day and time